### ISSUE TITLE
What is the specific difference between controlnet_inpainting and controlnet_inpainting_img2img? 

### ISSUE THREAD

**Issue by kelisiya** [haofanwang/ControlNet-for-Diffusers#21]
I tried to test the two algorithms but I feel that the effect is similar. Can I understand that the img2img version takes into account the image input allowed by the controlnet_inpainting , but controlnet_inpainting also support img .

**Comment by haofanwang** [comment]
Yes, there are very similar. [This function](https://github.com/haofanwang/ControlNet-for-Diffusers/blob/0fb5833a9990988e8f9b5db8508997447c7616cb/pipeline_stable_diffusion_controlnet_inpaint_img2img.py#L505) shows their difference.
